sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridClientTests.cs

```diff
@@ -58,6 +58,25 @@ public async Task CanPublishCloudEvent()
             await client.PublishCloudEventsAsync(GetCloudEventsList());
         }
 
+        [Test]
+        public async Task CanPublishEventUsingSAS()
+        {
+            EventGridClientOptions options = Recording.InstrumentClientOptions(new EventGridClientOptions());
```

Comment:
nit: move the options variable down to where it is used or just add it inline into the ctor.